src/AppBundle/Entity/Role.php

```diff
@@ -65,7 +66,7 @@ class Role extends AbstractPersonalTranslatable implements TranslatableInterface
      * @var Employee
      *
      * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Employee", inversedBy="roles")
-     * @Type("AppBundle\Entity\Employee")
+     * @Type("array")
```

Comment:
this return object in response, but in doc must be array. Swagger message in error
[https://github.com/geekhub-php/CheTheatre/blob/master/doc/theatre.json#L177](url)
change doc?